Framework/Core/src/ExpressionHelpers.h

```diff
@@ -10,18 +10,48 @@
 #ifndef O2_FRAMEWORK_EXPRESSIONS_HELPERS_H_
 #define O2_FRAMEWORK_EXPRESSIONS_HELPERS_H_
 #include "Framework/Expressions.h"
+#include "gandiva/filter.h"
+
 #include <vector>
 #include <iosfwd>
 
+using namespace arrow;
```

Comment:
In principle we should avoid `using namespace` in headers...

Reply:
It is an internal header, so I just saved myself some typing. I will remove the `using` statements when this is no longer WIP.